tests/FacebookTest.php

```diff
@@ -177,7 +177,7 @@ public function testPersistentDataHandlerCanBeForced()
     }
 
     /**
-     * @expectedException \InvalidArgumentException
+     * @expectedException PHPUnit_Framework_Error
```

Comment:
this will be broken on PHP 7 because the behavior for failed typehints is not the same anymore (it throws an Error instead of triggering a catchable fatal error)